js/dfpslotsprovider.js

```diff
@@ -197,6 +199,9 @@ export default class DFPSlotsProvider extends React.Component {
     let r = false;
     if (Object.keys(DFPManager.getRegisteredSlots()).length >= this.totalSlots) {
       DFPManager.removeListener('slotRegistered', this.loadAdsIfPossible);
+      if (this.props.limitedAds) {
+        DFPManager.configureLimitedAds(true);
```

Comment:
Unless you have a good reason to place this here, can you please move this call into the `applyConfigs` method? (see ~line 166)

Reply:
You're right, it makes more sense in applyConfigs !
Sorry, I should have spotted this

Done ✔️ => https://github.com/jaanauati/react-dfp/pull/209/commits/3a1de30221cdf3efb50132abab817770719ab611

Reply:
@jaanauati thanks for your guidance !

I hope it's OK for you, now 🙂

Reply:
Just tested it in my local and everything works well. Merging this stuff right now. New version will be released to npm as soon.

Thank you!.